server/filestore.go

```diff
@@ -909,6 +911,15 @@ func (fs *fileStore) recoverMsgs() error {
 		return err
 	}
 
+	// We had a bug that would leave fss files around during a snapshot.
+	// Clean them up here if we see them.
+	fms, err := filepath.Glob(path.Join(mdir, fssScanAll))
```

Comment:
Is the intent to return the error if filepath.Glob() produces an error?
Because you don't use fms outside and you WILL return that error as part of recoverMsgs(). So if you don't want to return an error due to Glob(), then I would recommend: `if fms, err := ...; err == nil && len(fms) > 0 {`

Reply:
Reading on filepath.Glob(), it says "Glob ignores file system errors such as I/O errors reading directories. The only possible returned error is ErrBadPattern, when pattern is malformed", so I guess it is fine the way you have it?

Reply:
I don't care about any errors here per so, correct.

Reply:
So that means you are going to make it a "one liner", right?

Reply:
Fixed.